tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+
+
+def test_proposal_source_and_items_file_references():
+    source_file = Path("docs/proposals/active/notation-sets/source.md")
+    items_file = Path("docs/proposals/active/notation-sets/ITEMS.md")
+    mdx_file = Path("proposals/notation-sets.mdx")
+
+    assert source_file.exists()
+    assert items_file.exists()
+    assert mdx_file.exists()
+
+    source_text = source_file.read_text()
```

Pylint (4.0.7)

[warning] Using open without explicitly specifying an encoding

(W1514)